judge-d-server/src/test/integration/com/hltech/judged/server/infrastructure/persistence/environment/JPAEnvironmentRepositoryIT.groovy

```diff
@@ -54,6 +54,35 @@ class JPAEnvironmentRepositoryIT extends Specification {
             }
     }
 
+    def 'should overwrite what was persisted before - one environment with multiple spaces'() {
+        given:
+        def firstSpace = new Space('a', [new ServiceId('serviceName1', 'serviceVersion1')] as Set)
+
+        and:
+        def secondSpace = new Space('b', [
+            new ServiceId('serviceName2', 'serviceVersion2'),
+            new ServiceId('serviceName3', 'serviceVersion3'),
+            new ServiceId('serviceName4', 'serviceVersion4')
+        ] as Set)
+
+        and:
+        def thirdSpace = new Space('def', [new ServiceId('serviceName5', 'serviceVersion5')] as Set)
+
+        and:
+        def environment = new Environment('environmentName', [firstSpace, secondSpace, thirdSpace] as Set)
```

Comment:
Minor - formatting - missing tabs

Reply:
once we have it without indent, once with. We would need to address this globally